### PR TITLE
Rebuild packages for missing metadata vol. 4

### PR DESCRIPTION
**Pull request by Chocimier**
Noarch only. Built and passed do_check on x86_64-musl. Nothing was tested manually whether it still works.

368 more packages to rebuild.

### CODE REVIEW

File: srcpkgs/albatross-themes/template [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+pkgname=albatross-themes
+version=1.7.4
+revision=2
 noarch=yes
-short_desc="An elegant black GTK2/3/Metacity/Xfwm theme"
 build_style=fetch
+depends="gtk+"
+short_desc="An elegant black GTK2/3/Metacity/Xfwm theme"
 maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/albatross-themes/template]: GPL-2.0-or-later

File: srcpkgs/artwiz-fonts/template [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-depends="font-util mkfontdir"
-font_dirs="/usr/share/fonts/artwiz-fonts"
+revision=4
 noarch=yes
 create_wrksrc=yes
+depends="font-util mkfontdir"
 short_desc="Small futuristic ASCII fonts for X"
 maintainer='Juan RP <xtraeme@voidlinux.eu>'
-homepage="http://artwizaleczapka.sourceforge.net/"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/artwiz-fonts/template]: GPL-2.0-or-later

File: srcpkgs/eigen2/template [hunk 3]
```diff
@@ -2,7 +2,7 @@
 pkgname=eigen2
 reverts="3.2.2_1"
 version=2.0.17
-revision=1
+revision=2
 noarch=yes
 wrksrc="eigen-eigen-b23437e61a07"
 build_style=cmake
```
Comment (maxice8) [inline srcpkgs/eigen2/template]: GPL-3.0-or-later, LGPL-3.0-or-later

File: srcpkgs/flinks/template [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'flinks'
 pkgname=flinks
 version=0.4.3
-revision=2
+revision=3
 noarch=yes
 build_style=python2-module
 pycompile_module="flinkspkg"
```
Comment (maxice8) [inline srcpkgs/flinks/template]: GPL-3.0-or-later

File: srcpkgs/gespeaker/template [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'gespeaker'
 pkgname=gespeaker
 version=0.8.6
-revision=1
+revision=2
 noarch=yes
 build_style=python2-module
 pycompile_dirs="/usr/share/gespeaker"
```
Comment (maxice8) [inline srcpkgs/gespeaker/template]: GPL-2.0-or-later

File: srcpkgs/guilt/template [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
-revision=1
+revision=2
 _githash=22d785dd24329170f66e7696da38b3e90e033d61
 _gitshort="${_githash:0:7}"
-wrksrc="guilt-${_gitshort}"
 noarch=yes
+wrksrc="guilt-${_gitshort}"
 build_style=gnu-makefile
 make_build_args="-C Documentation"
 make_install_args="mandir=/usr/share/man install install-doc"
```
Comment (maxice8) [inline srcpkgs/guilt/template]: GPL-2.0-only

File: srcpkgs/indic-otf/template [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
+noarch=yes
 wrksrc=indic
 build_wrksrc=OpenType
 makedepends="font-util xbps-triggers"
 depends="${makedepends}"
-noarch="yes"
-font_dirs="/usr/share/fonts/TTF"
 short_desc="Indic OpenType fonts collection"
 maintainer="Duncaen <duncaen@voidlinux.eu>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/indic-otf/template]: GPL-2.0-or-later

File: srcpkgs/liberation-fonts-ttf/template [hunk 8]
```diff
@@ -1,17 +1,17 @@
...
-revision=1
+revision=2
+noarch=yes
 makedepends="font-util xbps-triggers"
 depends="${makedepends}"
-noarch="yes"
-font_dirs="/usr/share/fonts/TTF"
 short_desc="TrueType fonts from RedHat"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2, ${pkgname}"
```
Comment (maxice8) [inline srcpkgs/liberation-fonts-ttf/template]: OFL-1.1

File: srcpkgs/libquvi-scripts/template [hunk 9]
```diff
@@ -1,8 +1,8 @@
 # Template file for 'libquvi-scripts'
 pkgname=libquvi-scripts
 version=0.4.21
-revision=1
-noarch="yes"
+revision=2
+noarch=yes
 build_style=gnu-configure
 hostmakedepends="pkg-config lua"
 makedepends="lua"
```
Comment (maxice8) [inline srcpkgs/libquvi-scripts/template]: LGPL-2.1-or-later

File: srcpkgs/mailpile/template [hunk 10]
```diff
@@ -1,17 +1,22 @@
...
+wrksrc="Mailpile-${version}"
 build_style=python2-module
+pycompile_module="static mailpile"
 hostmakedepends="python-setuptools"
 depends="python python-Jinja2 python-lxml python-MarkupSafe python-Pillow
  python-spambayes python-pydns python-pgpdump gnupg python-setuptools"
-pycompile_module="static $pkgname"
 short_desc="Modern email client with user-friendly encryption and privacy features"
 maintainer="Duncaen <duncaen@voidlinux.eu>"
 license="AGPL-3, Apache-2.0"
```
Comment (maxice8) [inline srcpkgs/mailpile/template]: AGPL-3.0-or-later

File: srcpkgs/makepasswd/template [hunk 11]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'makepasswd'
 pkgname=makepasswd
 version=1.10.9
-revision=1
+revision=2
 noarch=yes
 depends="perl-Crypt-OpenSSL-Random perl-Crypt-PasswdMD5"
 short_desc="Create and/or encrypt true-random-seeded password strings"
```
Comment (maxice8) [inline srcpkgs/makepasswd/template]: GPL-2.0-or-later

File: srcpkgs/mimms/template [hunk 12]
```diff
@@ -1,9 +1,9 @@
...
 pkgname=mimms
 version=3.2.1
-revision=2
-build_style=python2-module
+revision=3
 noarch=yes
+build_style=python2-module
 pycompile_module="libmimms"
 hostmakedepends="python-setuptools"
 depends="libmms python"
```
Comment (maxice8) [inline srcpkgs/mimms/template]: GPL-3.0-or-later

File: srcpkgs/mkinitcpio-encryptssh/template [hunk 13]
```diff
@@ -1,16 +1,16 @@
 # Template file for 'mkinitcpio-encryptssh'
 pkgname=mkinitcpio-encryptssh
 version=0.16
```
Comment (maxice8) [inline srcpkgs/mkinitcpio-encryptssh/template]: https://github.com/grazzolini/dropbear_initrd_encrypt

This hook was split into 3 different ones, is there any reason to keep it @thypon  ?

File: srcpkgs/mongroup/template [hunk 14]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'mongroup'
 pkgname=mongroup
 version=0.4.1
-revision=1
+revision=2
+noarch=yes
+depends="mon"
 short_desc="Monitor a group of processes with mon"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/mongroup/template]: This one doesn't actually have a license listed.

So it is all rights reserved and propietary.

Arch Linux uses MIT for it but not MIT license is present.

@the-maldridge @Vaelatern

File: srcpkgs/ohsnap-font/template [hunk 15]
```diff
@@ -1,18 +1,18 @@
...
-noarch="yes"
+revision=2
+noarch=yes
+wrksrc="ohsnap-${version}"
 makedepends="font-util"
 depends="${makedepends}"
-font_dirs="/usr/share/fonts/misc"
 short_desc="Monospaced font based on artwiz snap"
 maintainer='Juan RP <xtraeme@voidlinux.eu>'
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/ohsnap-font/template]: GPL-2.0-only

File: srcpkgs/python-functools32/template [hunk 16]
```diff
@@ -1,16 +1,16 @@
...
 wrksrc="functools32-${version}-1"
 build_style=python2-module
+pycompile_module="functools32"
 hostmakedepends="python-setuptools"
 depends="python"
-pycompile_module="functools32"
 short_desc="Backport of the functools module from Python 3.2"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
-homepage="https://github.com/MiCHiLU/python-functools32"
 license="PSF"
```
Comment (maxice8) [inline srcpkgs/python-functools32/template]: Python-2.0

File: srcpkgs/python-pydns/template [hunk 17]
```diff
@@ -1,13 +1,13 @@
...
 noarch=yes
+wrksrc="${pkgname#*-}-${version}"
 build_style=python2-module
+pycompile_module="DNS"
 hostmakedepends="python-setuptools"
 depends="python"
-pycompile_module="DNS"
 short_desc="Python DNS library"
 maintainer="Duncaen <mail@duncano.de>"
 license="CNRI"
```
Comment (maxice8) [inline srcpkgs/python-pydns/template]: CNRI-Python

**Comment by maxice8** [comment]
My review is concluded, 3 packages were proposed to be removed:

ls++-git -> Git package with no updates for 4 years
yt-play -> Git package with no updates for 4 years
mongroup -> No license file, no copyright disclaimer, it is effectively propietary

File: srcpkgs/mkinitcpio-encryptssh/template (hunk 13 above)
Comment (Chocimier) [reply srcpkgs/mkinitcpio-encryptssh/template]: Any change needed?

**Comment by maxice8** [comment]
On Sat, Nov 17, 2018 at 08:39:42AM -0800, Piotr wrote:
> Chocimier commented on this pull request.
> 
> 
> 
> > @@ -1,16 +1,16 @@
>  # Template file for 'mkinitcpio-encryptssh'
>  pkgname=mkinitcpio-encryptssh
>  version=0.16
> 
> Any change needed?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/4800#discussion_r234414008

Not yet.

**Comment by maxice8** [comment]
Merged

**Pull request closed without merging**